diff --git a/node_modules/mp-colorui/dist/index.esm.js b/node_modules/mp-colorui/dist/index.esm.js
index eb78386..8d85135 100644
--- a/node_modules/mp-colorui/dist/index.esm.js
+++ b/node_modules/mp-colorui/dist/index.esm.js
@@ -1,5 +1,6 @@
 import React, { useState as useState$1, useEffect as useEffect$1, Component as Component$1, useMemo as useMemo$1, useRef as useRef$1, createContext as createContext$1 } from 'react';
-import Taro, { pxTransform, useReady } from '@tarojs/taro';
+import { pxTransform, useReady } from '@tarojs/taro';
+import * as Taro from '@tarojs/taro';
 import { View, Image, Text, Button, Checkbox, CheckboxGroup, ScrollView, Input, Block, Radio, RadioGroup, PickerView, PickerViewColumn, Picker, Swiper, SwiperItem, Video, Switch, Textarea } from '@tarojs/components';
 
 /*! *****************************************************************************
@@ -347,7 +348,7 @@ var fullDate = function (num) {
 var getNowDate = function (type) {
     var date = new Date();
     var year = date.getFullYear();
-    var month = fullDate(date.getMonth());
+    var month = fullDate(date.getMonth() + 1);
     var day = fullDate(date.getDate());
     if (type === "year")
         return "" + year;
@@ -5683,7 +5684,7 @@ function ClMessage(props) {
                         clickClose();
                         clearTimeout(timer);
                         timer = null;
-                        props.message = "";
+                        setTempMessage("");
                     }, durationTime * 1000);
                 }
                 setContentHeight(tempHeight);
@@ -5693,7 +5694,7 @@ function ClMessage(props) {
     }, [showMessage]);
     useEffect$1(function () {
         show && setShowMessage(!!show);
-        props.show = false;
+        setShowMessage(false);
     }, [show]);
     useEffect$1(function () {
         if (show) {
diff --git a/node_modules/mp-colorui/lib/src/components/message/index.js b/node_modules/mp-colorui/lib/src/components/message/index.js
index b485432..f757698 100644
--- a/node_modules/mp-colorui/lib/src/components/message/index.js
+++ b/node_modules/mp-colorui/lib/src/components/message/index.js
@@ -73,7 +73,7 @@ export default function ClMessage(props) {
                         clickClose();
                         clearTimeout(timer);
                         timer = null;
-                        props.message = "";
+                        setTempMessage("");
                     }, durationTime * 1000);
                 }
                 setContentHeight(tempHeight);
@@ -83,7 +83,7 @@ export default function ClMessage(props) {
     }, [showMessage]);
     useEffect(() => {
         show && setShowMessage(!!show);
-        props.show = false;
+        setShowMessage(false)
     }, [show]);
     useEffect(() => {
         if (show) {
diff --git a/node_modules/mp-colorui/lib/src/lib/index.js b/node_modules/mp-colorui/lib/src/lib/index.js
index 54891a2..f6b9797 100644
--- a/node_modules/mp-colorui/lib/src/lib/index.js
+++ b/node_modules/mp-colorui/lib/src/lib/index.js
@@ -15,7 +15,7 @@ const fullDate = (num) => {
 export const getNowDate = (type) => {
     const date = new Date();
     const year = date.getFullYear();
-    const month = fullDate(date.getMonth());
+    const month = fullDate(date.getMonth() + 1);
     const day = fullDate(date.getDate());
     if (type === "year")
         return `${year}`;
